forms-flow-api/src/formsflow_api/resources/formio.py

```diff
@@ -26,6 +26,25 @@
 
 API = Namespace("Formio", description="formio")
 
+role = API.model(
```

Comment:
Yes, this is correct. We need to add this to all of the endpoints, not just roles endpoint.